run/docker/proxy/proxy.conf

```diff
@@ -1,15 +1,17 @@
 <VirtualHost *:80>
+  # Frontend
```

Comment:
I see you've moved this higher up so that it catches the `/openmrs/spa` before `/openmrs` . Got it now!